rosserial_tivac/package.xml

```diff
@@ -0,0 +1,27 @@
+<package>
+  <name>rosserial_tivac</name>
+  <version>0.0.1</version>
+  <description>
+  rosserial_tivac package provides the required hardware definitions for compiling rosserial_client targets for TivaC Launchpad 
+  evaluation boards.
+  </description>
+  <maintainer email="vmatos@robosavvy.com">Vitor Matos</maintainer>
+  <license>BSD</license>
+  <url type="website">http://wiki.ros.org/rosserial_tivac</url>
+  <url type="repository">https://github.com/robosavvy/rosserial</url>
+  <author>Vitor Matos</author>
+  <author email="vmatos@robosavvy.com">Vitor Matos</author>
+  
+  
+  <maintainer email="vmatos.pt@gmail.com">Vitor Matos</maintainer>
+  <license>BSD</license>
+  <url>http://ros.org/wiki/rosserial_tivac</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <run_depend>rospy</run_depend>
```

Comment:
Good question.
I based the development of this package on rosserial_arduino for good practices.
I guess the run depend is there for make_library script rospkg import.